WtWSMS/localization/english/dynasties/symmachus_dynasty_updates_l_english.yml

```diff
@@ -0,0 +1,286 @@
+dynn_Comisius: "Comisius"
+dynn_Fabullius: "Fabullius"
+dynn_Rabilius: "Rabilius"
+dynn_Autronius: "Autronius"
+dynn_Verginius: "Verginius"
+dynn_Laberius: "Laberius"
+dynn_Sallienus: "Sallienus"
+dynn_Careius: "Careius"
+dynn_Popilius: "Popilius"
+dynn_Masculinius: "Masculinius"
+dynn_Loreius: "Loreius"
+dynn_Sentius: "Sentius"
+dynn_Papirius: "Papirius"
+dynn_Fecenius: "Fecenius"
+dynn_Statilius: "Statilius"
+dynn_Tarquitius: "Tarquitius"
+dynn_Magnius: "Magnius"
+dynn_Pullius: "Pullius"
+dynn_Ceccius: "Ceccius"
+dynn_Allius: "Allius"
+dynn_Lucilius: "Lucilius"
+dynn_Hetereius: "Hetereius"
+dynn_Trebellius: "Trebellius"
+dynn_Cossutius: "Cossutius"
+dynn_Rubrenus: "Rubrenus"
+dynn_Ateius: "Ateius"
+dynn_Carmaeus: "Carmaeus"
+dynn_Vicinius: "Vicinius"
+
+#Romano-Thracian dynasties
+dynn_Considius: "Considius"
+dynn_Veturius: "Veturius"
+dynn_Maetennius: "Maetennius"
+dynn_Cabellius: "Cabellius"
+dynn_Anconteius: "Anconteius"
+dynn_Nolanius: "Nolanius"
+dynn_Marrenus: "Marrenus"
+dynn_Rascanius: "Rascanius"
+dynn_Samacius: "Samacius"
+dynn_Pacuvius: "Pacuvius"
+dynn_Fabricius: "Fabricius"
+dynn_Aufonius: "Aufonius"
+dynn_Sippius: "Sippius"
+dynn_Cassicius: "Cassicius"
+dynn_Villanius: "Villanius"
+dynn_Tironius: "Tironius"
+dynn_Sentilius: "Sentilius"
+dynn_Genucius: "Genucius"
+dynn_Disius: "Disius"
+dynn_Canuleius: "Canuleius"
+dynn_Nemonius: "Nemonius"
+dynn_Ersidius: "Ersidius"
+dynn_Truccius: "Truccius"
+dynn_Catonius: "Catonius"
+dynn_Barbius: "Barbius"
+dynn_Aburius: "Aburius"
+dynn_Numitorius: "Numitorius"
+dynn_Manneius: "Manneius"
+dynn_Uttiedius: "Uttiedius"
+dynn_Titovius: "Titovius"
+dynn_Vetidius: "Vetidius"
+dynn_Carullius: "Carullius"
+dynn_Scurricius: "Scurricius"
+dynn_Aprilius: "Aprilius"
+
+#Romano-Illyrian dynasties
+dynn_Artorius: "Artorius"
+dynn_Nerfinius: "Nerfinius"
+dynn_Barbaricius: "Barbaricius"
+dynn_Dirutius: "Dirutius"
+dynn_Varenus: "Varenus"
+dynn_Garinius: "Garinius"
+dynn_Ligustius: "Ligustius"
+dynn_Obultronius: "Obultronius"
+dynn_Tatusius: "Tatusius"
+dynn_Saufeius: "Saufeius"
+dynn_Murcidius: "Murcidius"
+dynn_Tarius: "Tarius"
+dynn_Eupunius: "Eupunius"
+dynn_Feredius: "Feredius"
+dynn_Urgulanius: "Urgulanius"
+dynn_Palpellius: "Palpellius"
+dynn_Crispuleius: "Crispuleius"
+dynn_Florius: "Florius"
+dynn_Veiedius: "Veiedius"
+dynn_Subulnius: "Subulnius"
+dynn_Aebutius: "Aebutius"
+dynn_Baebidius: "Baebidius"
+dynn_Mescenius: "Mescenius"
+dynn_Tarconius: "Tarconius"
+dynn_Utius: "Utius"
+dynn_Billianius: "Billianius"
+dynn_Titusidius: "Titusidius"
+dynn_Ancharenus: "Ancharenus"
+dynn_Camusenus: "Camusenus"
+dynn_Aprofennius: "Aprofennius"
+dynn_Quesidius: "Quesidius"
+dynn_Aeronius: "Aeronius"
+dynn_Vesidius: "Vesidius"
+dynn_Axenna: "Axenna"
+
+#Vandal dynasties
+dynn_Victohali: "Victohali"
+dynn_Carini: "Carini"
+dynn_Lugii: "Lugii"
+dynn_Lacringi: "Lacringi"
+dynn_Harii: "Harii"
+dynn_Helveconae: "Helveconae"
+dynn_Manimi: "Manimi"
+dynn_Helisii: "Helisii"
+dynn_Nahanarvali: "Nahanarvali"
```

Comment:
> This is Latin spelling, but in Vandal I believe the ending "i" or "ae" used to indicate plural should be removed?

Honestly, I don't know. I thought about removing the Latin endings, but in the end decided to stick with them as they are more traditional.

Reply:
They are more traditional in Latin literature because there are Latin versions, but the Vandals being Romanised to that extent compared to other similar East Germanic cultures doesn't make much sense

Reply:
> They are more traditional in Latin literature because there are Latin versions, but the Vandals being Romanised to that extent compared to other similar East Germanic cultures doesn't make much sense

Ok, I've Germanized the namelist based on Gothic, the closest one to Vandalic. I also replaced tribal dynasties with patronymic ones (ending with -igg, i.e. ing in Gothic language).